### ISSUE TITLE
t2i_workflow.json and ti2i_workflow.json cannot match code, please fix is.

### ISSUE THREAD

**Issue by yizhangliu** [AIFSH/OmniGen-ComfyUI#19]
t2i_workflow.json and ti2i_workflow.json cannot match code, please fix is.

**Comment by set-soft** [comment]
ti2i connects an image to the latent (?!)
And when I tried entering a latent there it was interpreted as an input image ...
Also the node for text: Why not using one of the available text or string boxes floating around?

**Comment by yegods** [comment]
yeah these workflows don't actually function.  a bit annoying and frustrating.

**Comment by set-soft** [comment]
> yeah these workflows don't actually function. a bit annoying and frustrating.

Don't get frustrated, just use any of the alternative nodes!
Like: https://github.com/chflame163/ComfyUI_OmniGen_Wrapper
I'm trying to modify it to play nice with Comfy_UI, i.e. use the memory management and data types from the core, but is WIP: https://github.com/set-soft/ComfyUI_OmniGen_Wrapper
In the "vae" branch I'm moving the VAE decode outside, eliminating temporal files, adding negative prompt and I'll use the external VAE to encode (next step).

Note: this model is funny, but is slow and doesn't produce very good images, but you can use its output as base for a Flux image to image workflow.

**Comment by mhnoni** [comment]
> > yeah these workflows don't actually function. a bit annoying and frustrating.
> 
> Don't get frustrated, just use any of the alternative nodes! Like: https://github.com/chflame163/ComfyUI_OmniGen_Wrapper I'm trying to modify it to play nice with Comfy_UI, i.e. use the memory management and data types from the core, but is WIP: https://github.com/set-soft/ComfyUI_OmniGen_Wrapper In the "vae" branch I'm moving the VAE decode outside, eliminating temporal files, adding negative prompt and I'll use the external VAE to encode (next step).
> 
> Note: this model is funny, but is slow and doesn't produce very good images, but you can use its output as base for a Flux image to image workflow.

Hi Salvador
Any chance you could fix your ComfyUI_OmniGen_Wrapper? looks like we can't install it anymore from the manager.

**Comment by set-soft** [comment]
Hi @mhnoni !

> Any chance you could fix your ComfyUI_OmniGen_Wrapper? looks like we can't install it anymore from the manager.

Did you try using the code from  https://github.com/set-soft/ComfyUI_OmniGen_Wrapper ?  I even installed it from the ComfyUI manager (relaxing security to weak). It works, well ... for more than 16 GB of system RAM (and 8 GB of VRAM), with just 16 GB of system RAM it takes forever to load the model.

**Comment by mhnoni** [comment]
> Hi [@mhnoni](https://github.com/mhnoni) !
> 
> > Any chance you could fix your ComfyUI_OmniGen_Wrapper? looks like we can't install it anymore from the manager.
> 
> Did you try using the code from https://github.com/set-soft/ComfyUI_OmniGen_Wrapper ? I even installed it from the ComfyUI manager (relaxing security to weak). It works, well ... for more than 16 GB of system RAM (and 8 GB of VRAM), with just 16 GB of system RAM it takes forever to load the model.

Yes I did, I installed "ComfyUI_OmniGen_Nodes" by Salvador, successfully, but when I try to load the example workflow I still get missing nodes, and the manager can't find the missing one to install it, I installed every single node related to omni but still failed when I try the workflow "set-soft/ComfyUI_OmniGen_Nodes/tree/main/image)
/girl_demo_1024.json". 
And when I try "/set-soft/ComfyUI_OmniGen_Nodes/tree/main/image)
/girl_demo_512.png" I get different workflow that doesn't look like the one you shared on the main page.. 
Would you share your workflow? I could try again and see if I could make it work.

**Comment by set-soft** [comment]
Hi @mhnoni !
I tried using a fresh install and the first example: https://github.com/set-soft/ComfyUI_OmniGen_Nodes/blob/main/image/girl_demo_1024.png 
I got a missing node from Comfyroll, asked the manager to install it and the issue was solved.
Which nores are you missing?

**Comment by mhnoni** [comment]
> Hi [@mhnoni](https://github.com/mhnoni) ! I tried using a fresh install and the first example: https://github.com/set-soft/ComfyUI_OmniGen_Nodes/blob/main/image/girl_demo_1024.png I got a missing node from Comfyroll, asked the manager to install it and the issue was solved. Which nores are you missing?

Hi 
Sorry about the late replay. I will try to reinstall the comfy and see if it works since I formatted my pc.
